### ISSUE TITLE
Deeplearning?

### ISSUE THREAD

**Issue by tvorogme** [tvorogme/scala#2]
Add [Deeplearning.scala](https://github.com/ThoughtWorksInc/DeepLearning.scala) tutorial.

- [ ] Simple titanic. (neural network with softmax categorical prediction)
- [ ] Cats and dogs. (convolution neural network on photos of dogs and cats)

Maybe:
Model Zoo
